.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["@babel/preset-env", "@babel/preset-react"]
+}
```

Comment:
Looks like there needs to be a newline at the end of this file.

Reply:
Good catch!